sharepoint/sharepoint-ps/sharepoint-online/Set-SPOAppPrioritizationPolicy.md

```diff
@@ -92,7 +92,7 @@ Accept wildcard characters: False
 
 ### -QuotaMultiplier
  
-This parameter specifies the quota multiplier limit for the scaling feature. Value must be between 1 and 10.
+This parameter specifies the quota multiplier limit for the scaling feature. Value must be between 2 and 10.
```

Comment:
@killerewok2000 this means the backend code also needs update? As I think it's still doing 1-10